### DISCUSSION TITLE
Next-intl translation are not working

### DISCUSSION THREAD

**Discussion by mohitrakh** [amannn/next-intl#1543] [Q&A]
### Description

On my login page the translation of navbar is not working after refresh it
when i navigate to different page and then come on the login it works
but when i refresh the page it stops working for some reason i don't understand
I get hydration error on the login page as well 
the thing is when removed the header and check the login page it did not had any hydration error as well
so issue is in the header component but header works completely fine on other page and translation works as great as well
it just than in the login page it stops 

### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on [GitHub](https://github.com/amannn/next-intl/issues) and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

http://localhost:3001/login

### Reproduction description

Steps to reproduce:
1. Open reproduction
2. Click on …
3. See error: …

i cant give the website as it is confidential

### Expected behaviour

I want the header to be translated on every page

**Comment by amannn** [comment]
Sorry, I don't know either and also can't have a look since there's no reproduction.

I'll move this to a discussion since it looks like a usage question.
